package.json

```diff
@@ -20,6 +20,7 @@
     "react-datepicker": "^1.6.0",
     "react-dropzone": "^5.0.1",
     "react-redux": "^5.0.7",
+    "react-select": "^2.3.0",
```

Comment:
Please add it to dependencies section in Readme as well with an explanation in comment

Reply:
Added